installer/windows/scripts/configure_acrylic.bat

```diff
@@ -6,28 +6,35 @@ ECHO Executing configure_acrylic.bat
 
 SETLOCAL ENABLEDELAYEDEXPANSION
 
-SET "acrylic_config=C:\Program Files (x86)\Acrylic DNS Proxy\AcrylicHosts.txt"
+SET "acrylic_config_host=C:\Program Files (x86)\Acrylic DNS Proxy\AcrylicHosts.txt"
+SET "acrylic_configuration=C:\Program Files (x86)\Acrylic DNS Proxy\AcrylicConfiguration.ini"
 
 :: If it doesn't exist, create it
-if not exist "%acrylic_config%" (
+if not exist "%acrylic_config_host%" (
     :: Create the file
-    ECHO # Acrylic DNS Proxy configuration file > "%acrylic_config%"
-    ECHO # For more information please visit http://mayakron.altervista.org >> "%acrylic_config%"
-    ECHO # >> "%acrylic_config%"
+    ECHO # Acrylic DNS Proxy configuration file > "%acrylic_config_host%"
+    ECHO # For more information please visit http://mayakron.altervista.org >> "%acrylic_config_host%"
+    ECHO # >> "%acrylic_config_host%"
 )
 
 :: Check if the TLD is already in the file
-FINDSTR /c:".massa" "%acrylic_config%" >nul 2>&1
+FINDSTR /c:".massa" "%acrylic_config_host%" >nul 2>&1
 if %errorlevel%==0 (
     ECHO TLD already in the file
     EXIT 0
 )
 
+:: change LocalIPv4BindingAddress to 127.0.0.1
+FINDSTR /V "LocalIPv4BindingAddress" "%acrylic_configuration%" > temp.txt
+MOVE /Y temp.txt "%acrylic_configuration%"
+ECHO. >> "%acrylic_configuration%"
+ECHO LocalIPv4BindingAddress=127.0.0.1 >> "%acrylic_configuration%"
+
```

Comment:
IMO, this should be before the TLD check. Otherwise it will not be done if the TLD is already set, but might be required in some cases, such as if the user had a previous version of MassaStation that didn't performed that change.

Reply:
good catch, for me it's blocking, we must to it before merging